### DISCUSSION TITLE
python-can (Windows)

### DISCUSSION THREAD

**Discussion by vondraussen** [jgressmann/supercan#13] [Q&A] [answered]
I guess on Linux supercan can be used via socketcan with python-can.
Is it possible to use supercan with python-can on Windows?

**Comment by jgressmann** [comment]
Yes to the Linux part. Not that I know of to the Windows part.

**Comment by jgressmann** [comment, marked as answer]
[Release v1.0.24](https://github.com/jgressmann/supercan/releases/tag/v1.0.24) adds support for [python-can](https://python-can.readthedocs.io/en/stable/).
